tests/conftest.py

```diff
@@ -58,41 +57,6 @@ def db_engine():
     return engine
 
 
```

Comment:
The fixtures for the unit and func tests have to be different. We can't roll back the transaction everytime, because we need things to _really_ reach the DB in the func tests.